wordpress-mobile/WordPress-iOS
# [Stats Refresh] Insights Most Popular Time: update to new design

Fixes #11909 

This updates Insights _Most Popular Time_ to use the two-column stat card.

To test:

- On a site with no Most Popular data, go to Insights _Most Popular Time_.
  - Verify `No data yet` is shown.
- On a site with Most Popular, go to Insights _Most Popular Time_.
  - Verify the data is shown in two columns.

Update release notes:


---
<img width="400" alt="most_popular_no_data" src="https://user-images.githubusercontent.com/1816888/59636289-62868c80-9110-11e9-86b3-074cce315ed4.png">

---
<img width="400" alt="most_popular" src="https://user-images.githubusercontent.com/1816888/59636334-7c27d400-9110-11e9-80f7-51ebaa3927e4.png">
